src/Illuminate/Foundation/Bootstrap/HandleExceptions.php

```diff
@@ -83,7 +83,7 @@ public function handleException($e)
             //
         }
 
-        if ($this->app->runningInConsole()) {
+        if ($this->app->runningInConsole() or $this->app->runningInConsoleAsHttpServer()) {
```

Comment:
If you add the condition into `runningInConsole()` this file does not need to change.